content/learn/book/migration-guides/0.6-0.7/_index.md

```diff
@@ -0,0 +1,211 @@
+
+### Remove face_toward.rs
+
+<https://github.com/bevyengine/bevy/pull/4277>
+
+The `FaceToward` trait was removed. To migrate, replace every occurrence of `Mat4::face_toward` to `Mat4::look_at_rh`.
+
+### `World::entities_mut` is now unsafe
+
+<https://github.com/bevyengine/bevy/pull/4093>
+
+```rs
+// 0.6
+world.entities_mut()
+
+// 0.7
+unsafe { world.entities_mut() }
+```
+
+### Mesh vertex buffer layouts
+
+<https://github.com/bevyengine/bevy/pull/3959>
+
+TODO
+
+### Remove the need for 'IntoSystem::into_system()' when using run criteria piping
+
+<https://github.com/bevyengine/bevy/pull/3923>
+
+```rs
+// 0.6
+.with_run_criteria(RunCriteria::pipe(
+    "is_done_label",
+    IntoSystem::into_system(inverse),
+))
+
+// 0.7
+.with_run_criteria(RunCriteria::pipe("is_done_label", inverse))
+```
+
+### Obviate the need for RunSystem, and remove it
+
+<https://github.com/bevyengine/bevy/pull/3817>
+
+TODO
+
+### Replace VSync with PresentMode
+
+https://github.com/bevyengine/bevy/pull/3812
+
+Instead of using a boolean flag for vsync we switched to using a [`PresentMode`] enum with multiple variants.
+
+```rs
+// 0.6
+App::new()
+    .insert_resource(WindowDescriptor {
+        vsync: false,
+        ..Default::default()
+    })
+
+// 0.7
+App::new()
+    .insert_resource(WindowDescriptor {
+        present_mode: PresentMode::Immediate,
+        ..Default::default()
+    })
+```
+
+<!-- TODO update to 0.7 link -->
+[`PresentMode`]: http://dev-docs.bevyengine.org/bevy/window/enum.PresentMode.html
+
+### Fix mul_vec3 tranformation order
+
+<https://github.com/bevyengine/bevy/pull/3811>
+
+Transforms are now consistently applied in the standard scale -> rotate -> translate. This doesn't require any code changes, but it means SpriteBundle will behave as expected when rotating.
+
+### Use marker components for cameras instead of name strings
+
+<https://github.com/bevyengine/bevy/pull/3635>
+
+TODO
+
+### Remove the config api
+
+<https://github.com/bevyengine/bevy/pull/3633>
+
+TODO
+
+### Cameras now point at RenderTarget rather than Window
+
+<https://github.com/bevyengine/bevy/pull/3412>
+
+This change was made to support rendering to textures. Users working with multiple windows may be affected.
+
+```rs
+// 0.6
+commands.spawn_bundle(PerspectiveCameraBundle {
+    camera: Camera {
+        window: window_id,
+        ..Default::default()
+    },
+    ..Default::default()
+});
+
+// 0.7
+commands.spawn_bundle(PerspectiveCameraBundle {
+    camera: Camera {
+        target: RenderTarget::Window(window_id),
+        ..Default::default()
+    },
+    ..Default::default()
+});
+```
+
+### Implement init_resource for Commands and World
+
+<https://github.com/bevyengine/bevy/pull/3079>
+
+```rs
+#[derive(Default)]
+struct Scoreboard {
+    current_score: u32,
+    high_score: u32,
+}
+
+// 0.6
+commands.insert_resource(Scoreboard::Default());
+
+// 0.7
+commands.init_resource::<Scoreboard>();
```

Comment:
Don't the 0.6 and the 0.7 examples here have different behaviour in case there is already a scoreboard resource?

Maybe we should just say this is new functionality on Commands, so no 0.6 example.

Reply:
Well, it's a migration guide, if it's purely new then it doesn't need any migration. I just did that after a quick glance at the PR. I wasn't sure what the change was.